nss-day-cohort-39/nutshell-smaragdine-sugar-gliders
# Add Friends From Chat Component

**Changes**
- Added logic to allow users to add friends by clicking names in the chat component

**Testing** 
- Serve the application to localhost 
- Login or register as a new user
- Click a user's name in the chat box that is not currently a friend and verify that the "add friend" confirmation alert appears, then proceed to add friend. 
- Click a user's name that is already a friend and verify that the "already a friend" alert displays.
